### PR TITLE
e

### PR DESCRIPTION
**Pull request by Fen0l**

### CODE REVIEW

**Comment by fernandog** [comment]
@JohnDooe use AlphaRatio as base for example. see log messages and the needed commands in doSearch like the name of variables, lists, the discarding torrent check and etc

**Comment by fernandog** [comment]
compare which def exists and which not. like the image def it's not necessary

File: sickbeard/providers/__init__.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
 from sickbeard.providers import btn, newznab, womble, thepiratebay, torrentleech, kat, iptorrents, torrentz, \
     omgwtfnzbs, scc, hdtorrents, torrentday, hdbits, hounddawgs, nextgen, speedcd, nyaatorrents, animenzb, bluetigers, cpasbien, fnt, xthor, torrentbytes, \
     freshontv, titansoftv, libertalia, morethantv, bitsoup, t411, tokyotoshokan, shazbat, rarbg, alpharatio, tntvillage, binsearch, torrentproject, extratorrent, \
```
Comment (duramato) [inline sickbeard/providers/__init__.py:28]: @JohnDooe This file cant be touched ,it should only inclued your provider..

Comment (Fen0l) [inline sickbeard/providers/__init__.py:28]: I dont use frequently git but at the beginning, it was something like that :) https://github.com/SickRage/SickRage/pull/118/files Dont know why it does that

Comment (duramato) [inline sickbeard/providers/__init__.py:28]: @JohnDooe the sickbeard/providers/**init**.py changed format ,so you need to update it

**Comment by miigotu** [comment]
The code in this provider is totally broken and would not behave properly. Without an account to use for grabbing html or some page source I can't even fix it up correctly.

Closing until someone can write it correctly or provide the necessary html or account.

**Pull request closed without merging**